### PR TITLE
Various Indexer/App fixes

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
- Fixed: {Mylar3) Indexer Host needs to include trailing /api 
- Fixed: Update AppIndexerRegex to support indexer ids over 100 
- Fixed Update AppIndexerRegex to support /api trailing (Mylar3)
#### Todos
- Tests  - we really should add app tests.  (how?)
-  Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
-  [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes mylar issue reported via discord

### CODE REVIEW

**Comment by Qstick** [comment]
Why do tests pass in Mylar without `/api`??????

image

Also, lets rework the regex to hit all 4 of these

image

**Comment by bakerboy448** [comment]
Asked OP on discord; we shall see

Earlier convo reporting the issue https://discord.com/channels/767843854736949299/767843854824374281/877193024424267777

Waiting for info https://discord.com/channels/767843854736949299/767843854824374281/877407136647356446

Will update the Regex in the AM; can split that into a different PR if that is easier?

**Comment by bakerboy448** [comment]
Regex Updated

Still no word on discord from the user who reported the Mylar issue

2BBCD739-7F47-4804-BA53-D04301C2E5D0

File: src/NzbDrone.Core/Applications/ApplicationBase.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ public abstract class ApplicationBase<TSettings> : IApplication
         protected readonly IAppIndexerMapService _appIndexerMapService;
         protected readonly Logger _logger;
 
-        protected static readonly Regex AppIndexerRegex = new Regex(@"\/(?<indexer>\d.)\/?$",
+        protected static readonly Regex AppIndexerRegex = new Regex(@"\/(?<indexer>\d{1,3})(?:\/(?:api)?\/?)?$",
```
Comment (Qstick) [inline src/NzbDrone.Core/Applications/ApplicationBase.cs:17]: Why not `\/(?<indexer>\d*)(?:\/(?:api)?\/?)?$`

Just incase someone has 1000 :D

**Review by Qstick — APPROVED** [review]

File: src/NzbDrone.Core/Applications/ApplicationBase.cs (hunk 1 above)
Comment (bakerboy448) [reply src/NzbDrone.Core/Applications/ApplicationBase.cs:17]: If they have 1000+ then they'll have other issues than indexers not mapping 😂

**Pull request merged**